docker-images/sw360populated/Dockerfile

```diff
@@ -14,7 +14,8 @@ COPY _deploy/liferay/* /opt/sw360/deploy/liferay/
 RUN set -ex \
  && for war in /opt/sw360/deploy/tomcat/*.war; do \
       folder=$(basename $war .war); \
-      mkdir -p /opt/sw360/tomcat-*/webapps/$folder; \
-      unzip -q $war -d /opt/sw360/tomcat-*/webapps/$folder; \
+      (cd /opt/sw360/tomcat-*; \
```

Comment:
This is an improvement in code quality, but should not change any functionality or logic, right?

Reply:
Yes.